benchmarks/scrolling/setup

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+for _ in {0..100000}; do
+    echo "y"
+done
```

Comment:
Can we use `printf` here, since we use it for everything already and it's a bit more portable?

Reply:
Yeah, good idea.